assets/components/prometheus-operator/manifests/templates/grafana/dashboards-1.14/persistentvolumesusage.yaml

```diff
@@ -533,7 +533,7 @@ data:
             ]
         },
         "time": {
-            "from": "now-7d",
```

Comment:
This seems opinionated, I'm not sure what this PR fixes exactly. For the storage usage, IMO having a long term overview is more important than seeing the change from the last few hours.

Reply:
It fixes this issue https://github.com/kinvolk/lokomotive/issues/587

Reply:
How does it fix it? It seems queries don't do any time aggregation, so I would expect it to work right after the metric shows up. Could you elaborate?

Reply:
How does this fix #587, is it because the 7-day dashboard doesn't have enough data because it only has data for 2 days?